packages/core/src/styles/typography/typography.stories.ts

```diff
@@ -131,7 +130,6 @@ export const legacyHeaders = () => {
 export const legacyParagraphs = () => {
   return html`
     <cds-demo cds-layout="vertical gap:lg">
-      <p cds-text="p0">The quick brown fox jumps over the lazy dog. (p0)</p>
```

Comment:
I took these out since they are not documented in clr-ui on the main website but can add them back if necessary.

Reply:
That's fine. I didn't know they weren't on the main website because I was working off the code in clr/ui